.github/workflows/build-container.yml

```diff
@@ -199,6 +199,19 @@ jobs:
       - name: Display logs
         run: make logs OPTS=''
 
+      - name: Login to quay.io (on release/tag create)
+        if: startsWith(github.ref, 'refs/tags/v')
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
```

Comment:
For example, you could test this PR, by forking this repo and replacing that registry name to a custom one (e.g.: docker, quay.io, ghcr.io, ...).

Reply:
That's because you appear as (read-only) outside collaborator. @caroav needs to add invite you to the Ceph org and nvmeof team:

![image](https://github.com/ceph/ceph-nvmeof/assets/37327689/cd9e27bf-6f39-4691-8d1d-bdf150c3abf2)

Reply:
@caroav , could you please invite me to the Ceph org and nvmeof team?